types/audit.d.ts

```diff
@@ -70,6 +70,28 @@ declare global {
       wastedPercent?: number;
     }
 
+    export interface ClientRectOverlapFailure  {
+      overlapScoreRatio: number;
+      tapTargetScore: number;
+      overlappingTargetScore: number;
+    }
+    
+    export interface TapTargetOverlapFailure extends ClientRectOverlapFailure {
+      tapTarget: Artifacts.TapTarget;
+      overlappingTarget: Artifacts.TapTarget;
+    }
+
+    export type TapTargetTableItem = {
```

Comment:
any reason this isn't an `interface`? can we make it an `interface`?

Reply:
The `makeTableDetails` items need to be `Array<{[x: string]: DetailsItem}>`, so we need to guarantee that the index is always a string. And I guess types automatically do that because they aren't extendable?

Anyway, I've changed it to an interface that extends the `DetailsRendererDetailsJSON` item type.